persistence/artist_repository.go

```diff
@@ -43,10 +43,13 @@ func (a *dbArtist) PostScan() error {
 	}
 	a.Artist.Stats = make(map[model.Role]model.ArtistStats)
 	for key, c := range stats {
-		if key == "total" {
-			a.Artist.Size = c["s"]
-			a.Artist.SongCount = int(c["m"])
-			a.Artist.AlbumCount = int(c["a"])
+		// Currently, Navidrome only returns albums for participants that are
+		// artist/album artist. Represent the size, song/album count as the max of
+		// these two fields
+		if key == "artist" || key == "albumartist" {
```

Comment:
This may affect the UI showing totals for when the role is unselected: https://github.com/navidrome/navidrome/blob/c795bcfcf7471c244b0735e990fe8ccd0252d0c8/ui/src/artist/ArtistList.jsx#L119 

Maybe a better approach is to create a new entry in the Stats (`max`?)